python/py-luma.emulator/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-luma.emulator
+github.setup        rm-hull luma.emulator 1.5.0
+revision            0
+categories          python
+maintainers         nomaintainer
+platforms           any
```

Comment:
as above, add `darwin`?

Reply:
As in https://github.com/macports/macports-ports/pull/20639#discussion_r1341875995%E2%80%94I don’t see a .so for this Python module.